docs/src/operate/configuration/proxy-configuration.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Proxy Configuration
+tags: [Operate, Proxy, MQTT, HTTP]
+description: Connecting %%te%% to a cloud via a proxy server
+---
+
+As of version 1.5.0, %%te%% can connect to a proxy server (for both HTTP and MQTT connections).
+
+The protocol for MQTT connections is HTTP CONNECT. This establishes an initial connection with HTTP, then connects via TCP as usual for MQTT.
+
+Proxy servers are not supported by the mosquitto bridge, so the built-in bridge must be enabled in order to connect %%te%% via a proxy server.
+
+By default, the configured proxy is used to connect to the cloud MQTT service and for all HTTP communication.
```

Comment:
```suggestion
By default, the configured proxy is used to connect to the cloud MQTT service and for all HTTP communications.
```

Reply:
`communication` sounds fine to me (though more of a native speaker gut feeling than anything).